### DISCUSSION TITLE
Welcome to gpt4free-ts Discussions!

### DISCUSSION THREAD

**Discussion by xiangsx** [xiangsx/gpt4free-ts#43] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NvH1986** [comment]
Im noob en try make smal steps for income make small start en no stress

**Reply by ch99000** [reply]
我想制作一个精美的梦幻画作

**Reply by NvH1986** [reply]
What u mean?

**Reply by Tony201273** [reply]
Hi guys I am Tony Edmunds and I am a complete newbie I have about 8 different product on github but I don't know where to start do any one have any advice for me yours sincerely Tony Edmunds

**Comment by Stayreal9944** [comment]
Hi

**Comment by Stayreal9944** [comment]
No response what's wrong

**Reply by jdgp-hub** [reply]
What do you not like?

**Comment by Dcee235** [comment]
Hello

**Comment by Deepali-1508** [comment]
Hii I'm Deepali , a passionate Frontend React Developer. My curiosity in A.I always wakes me  up.

**Comment by bitmelody** [comment]
Hi
Your GPT-GOD site is written in Chinese, so I can't even login.
https://gptgod.site/
Please help me with English version.
Thanks.

**Comment by doypro** [comment]
hello

**Comment by Madprojets** [comment]
Hey

**Comment by Tony201273** [comment]
@hi guys it's Tony Edmunds ere I am a complete newbie and I am having trouble finding my way around  I don't know what products are mine and I will appreciate your help and support yours sincerely Tony Edmunds
